### DISCUSSION TITLE
Zoneedit域名解析

### DISCUSSION THREAD

**Discussion by DavidLdf** [jeessy2/ddns-go#1207] [Ideas]
https://cp.zoneedit.com/support/dynamic/clients/jscript/conf.php/zoneedit-dynjs.html，这是域名解析官方的DDNS可是无法获取IPV6，请问大佬以后有计划可以提供对zoneedit域名解析服务器的支持吗？毕竟这也有免费的域名解析服务

**Comment by jeessy2** [comment]
试试callback
无计划支持，服务商太多了

**Reply by DavidLdf** [reply]
哦，谢谢答复
